### PR TITLE
chore: create twitch proxy tests

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Related to #490

This adds a few tests to ensure the Twitch Proxy data routes are still functioning correctly.

### CODE REVIEW

**Comment by a2937** [comment]
I don't understand. What's different between my local project and the one being hosted here. 

image

**Comment by a2937** [comment]
I need to see what exactly the results are from the workflow cause while it returns 200,something isn't matching up with my computer.

File: test/twitch-proxy.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// TODO: Remove once types for Jest are recognized
+/* eslint-disable no-undef */
+const axios = require('axios');
+const portMap = require('../port-map.json');
+const twitchProxyPort = portMap['twitch-proxy'];
+
+describe('kraken api', () => {
```
Comment (ShaunSHamilton) [inline test/twitch-proxy.test.js:7]: I feel silly for asking, but why/what is kraken relating to?

Comment (a2937) [reply test/twitch-proxy.test.js:7]: It refers to the version of Twitch API FreeCodeCamp is mocking. 

This file holds all of the [Kraken](https://github.com/freeCodeCamp/demo-projects/blob/main/apps/twitch-proxy/static-data/kraken.json) dataset.

File: test/jest-utils.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const baseUrl = port => {
+  if (process.env.DEMO_APPS_DOMAIN === 'localhost') {
+    const url = new URL('https://localhost');
```
Comment (ojeytonwilliams) [inline test/jest-utils.js:3]:
```suggestion
    const url = new URL('http://localhost');
```
This might resolve the test failures.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: test/jest-utils.js (hunk 2 above)
Comment (ShaunSHamilton) [reply test/jest-utils.js:3]: 🤫

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 

Thanks @a2937

**Review by ojeytonwilliams — APPROVED** [review]
Cheers, @a2937 this LGTM :+1:

**Pull request merged**